setup.py

```diff
@@ -24,46 +24,30 @@ def get_version(*file_paths):
     raise RuntimeError("Unable to find version string.")
 
 
-def get_requirements(requirements_file):
+def load_requirements(*requirements_paths):
```

Comment:
All the changes in this file are just cherry-picked from [cookiecutter-django-app](https://github.com/edx/cookiecutter-django-app/blob/098162bbd126b5a630741eec33fa4ae2071ecfeb/{{cookiecutter.repo_name}}/setup.py#L27) to bring it up to date and prevent weird issues parsing base.in.